added CSGO Inventory private check

LemonPAKA opened the pull request:
tested in csgo and it works, but didn't test in TF2,DOTA2 and portal2..I think it should work, because I didn't change any code about this part

LemonPAKA committed:
updated csgo inventory check feature

updated csgo inventory check feature, but still looking forward to anyone's better version...I am suffering for my trash codes

LemonPAKA committed:
added Simplified Chinese

added Simplified Chinese

LemonPAKA committed:
Update README.md

LemonPAKA committed:
updated csgo inventory check feature

updated csgo inventory check feature, but still looking forward to anyone's better version...I am suffering for my trash codes

rumblefrog reviewed (changes requested):
I can't review this PR as all the lines are overwritten, and I can't see what was changed. Can you fix this?

LemonPAKA committed:
restore to the original version

LemonPAKA committed:
trying to fix the overwritten problem

LemonPAKA commented:
I think I fixed the overwritten problem...

rumblefrog commented:
You should be force pushing the commit to the branch, the indentation changes are unnecessary and makes it harder for me to review.

rumblefrog commented on Anti_Private.sp:
```diff
@@ -278,55 +279,92 @@ void ParseProfile(const char[] sBody, int iClient)
-		GetClientAuthId(iClient, AuthId_SteamID64, SteamID, sizeof SteamID);
-	
-		if (STEAMWORKS_AVAILABLE())
+		if(GetEngineVersion()==Engine_CSGO)
```
Is there a reason why you are duplicating SteamWorks and SteamTools check just for CSGO.

rumblefrog commented on Anti_Private.sp:
```diff
@@ -278,55 +279,92 @@ void ParseProfile(const char[] sBody, int iClient)
 		switch (GetEngineVersion())
 		{
 			case Engine_TF2, Engine_DOTA, Engine_Portal2: {}
+			case Engine_CSGO:
```
The InventoryURL should be defined on start and engine be checked here

LemonPAKA committed:
restore again

LemonPAKA committed:
Simplified code and ..

Simplified code and InventoryURL sample has been added on start (would not actually use it), waiting for better solution

LemonPAKA committed:
restore again lmao

LemonPAKA committed:
used new API without time limit

used new API without time limit and Simplified new feature codes..

LemonPAKA commented:
I don't know how to defined InventoryURL as other games...so I just made a sample for it...and already pushed the commit to the branch

rumblefrog commented:
Yeah, I'm not sure why you pushing additional commits to restore and redo. I wanted you to do a **force push** to your branch.
